packages/components/src/Button/Button.tsx

```diff
@@ -11,16 +11,16 @@ const Button = ({
     disabled = false,
     extraAria = {},
     extraClasses = '',
-    size = 'large',
+    size = 'medium',
     title = '',
     type = 'primary',
 }: ButtonProps) => {
     const classes = createCssClassNames({
         'ids-btn': true,
-        [`ids-btn--${type}`]: !!type,
-        [`ids-btn--${size}`]: !!size,
+        [`ids-btn--${type}`]: true,
```

Comment:
as we use TypeScript both type and size accepts only specific string values, so both these conditions always be true